xml_inscriptions/metadata_only/KY.Lou.SAM.L.1929.17.727.xml

```diff
@@ -21,29 +21,28 @@
                <msContents>
                   <textLang mainLang="lat"/>
                   <msItem class="#funerary.epitaph">
-                     <p>Fragmentary epitaph</p>
+                     <p>fragmentary epitaph</p>
                   </msItem>
                </msContents>
                <physDesc>
                   <objectDesc ana="#fragment">
-                     <supportDesc ana="#stone.tufa">
+                     <supportDesc ana="#stone.other">
                         <support>
-                           <p>Fragment of coarse stone (possibly tufa)</p>
+                           <p>fragment of coarse stone (possibly tufa)</p>
                            <dimensions type="surface" unit="cm">
                               <height/>
                               <width/>
                               <depth/>
                            </dimensions>
-                           
                         </support>
                         <condition ana="#fragment.single">
                            <p>broken on left, right, and bottom edges</p>
                         </condition>
                      </supportDesc>
                      <layoutDesc>
-                        <layout columns="1" writtenLines="2">
-                              <!--use a <locus> to narrow down part of inscription being discussed, i.e. campus-->
-                              <p/>
+                        <layout writtenLines="2">
```

Comment:
I'd change columns to zero rather than removing, since the search may have issues finding it without it.